src/seismometer/data/pandas_helpers.py

```diff
@@ -140,6 +185,8 @@ def infer_label(dataframe: pd.DataFrame, label_col: str, time_col: str) -> pd.Da
         The column specifying the value to infer.
     time_col : time_col
         The time column associated with the value to infer.
+    impute_val : Optional[Number|str], optional
+        The value to impute for the label if timestamp exist, defaults to 1.
```

Comment:
Dumb question, would we ever want to impute a different value because of the timestamp? 

Like if no value is recorded for this event, use the prior value? 

I'm not sure what use cases we have right now for impute other than a flag to say "the event happened"

Reply:
I _think_ the answer is yes. At least, having a impute value for both scenarios makes sense to me.
However, I feel like that change is probably a bit out of scope of this PR and should instead be written up as a separate issue. My changes were only to fix the intended behavior, I didn't really modify to add to the functionality at all iirc.

Reply:
Ill write it up as an issue